lib/main.js

```diff
@@ -4,10 +4,11 @@
 import {Implementation} from './Implementation.js';
 import {implementerFiles} from '../implementations/index.js';
 
+export {Implementation};
+
 const keyValues = implementerFiles.map(
   implementation => [implementation.name, new Implementation(implementation)]);
-export const implementations = new Map(keyValues);
-const allImplementations = implementations;
+export const allImplementations = new Map(keyValues);
```

Comment:
Can we name this `implementations` instead of `allImplementations` so it doesn't become a breaking change?

Reply:
I prefer `implementations` too, so I've decided to just export both `implementations` and `allImplementations` until we do a major release and consensus on that naming convention occurs.